packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -231,7 +256,7 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 		localOpMetadata: unknown,
 	) {
 		const contents: unknown = this.serializer.decode(message.contents);
-		const { commit, sessionId } = this.messageCodec.decode(contents);
+		const { commit, sessionId } = this.messageCodec.decode(contents, {});
```

Comment:
why pass empty context here? do we know it will never be used?

Reply:
The empty context was just passed in to satisfy the codec updated with the new types, but I believe the decode function is schema-agnostic (documented in fieldBatchCodec of `/chunked-forest/codec/codec.ts`). But I now added a comment for clarification :)

Reply:
ya, I figured, it's just weird (maybe code smell?) that we know externally that this is true. I suppose it is optional :)